modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -172,9 +172,8 @@
      /* Master Tabs */
 
      // if no selected => select first
-     if ($('.master-tabs a[aria-selected]').length == 0) {
-         $('.master-tabs a:first').attr('aria-selected', 'true');
-         $('.master-tabs a:first').attr('tabindex', '0');
+     if ($('.master-tabs a[aria-selected]').length === || !== 0) {
```

Comment:
@ayumihamsaki just change all the doubles `==` to triples `===` (JS strict format).
Sorry if I missleaded you ;) 
Should be so: `if ($('.master-tabs a[aria-selected]').length === 0`

Reply:
lol, yah I wasn't sure when you wrote `||` if you meant or in English or not.

Reply:
@ayumihamsaki while you're still here, could you change all other places where you use non-strict compare `==` to strict `===`?